docs/features/shared-volumes-with-NFS.md

```diff
@@ -0,0 +1,129 @@
+1. “addr” – IP or fully qualified host name of server
+1. “args – args to mount remote dir.
+1. “path” – path on server to mount
+1. “type” – FS type of remote dir.
+1. “vol-path” – a path name that is relative to the mounted path abnove, volumes will be create relative to this path if non-null, if not specified then volumes are located under path above. The _vol-path_ entry allows the admin  admin to separate volumes for different docker service stacks within a single remote dir.
+1. "src" - indicates that the entry in the remote directory table refers to a parent entry in the same table. Combined with a _VolPath_ this allows the admin to manage placement of volumes on separate folders within a single remote dir. Especially useful when the admin wants to separate volumes for different service stacks into separate folders. It also allows the admin from having to list many duplicate entries, with the same remote directory details only to differ in the _VolPath_. For example consider two services _Svc_A_ and _Svc_B_ that are using volumes on a remote dir _NetShare_. To locate volumes for the two services in two separate directories under _NetShare_ the admin needs to create two duplicate entries with the same details with different _VolPath_ values. A change in one will need to be duplicated in the other as well. Instead the admin creates a single entry for _NetShare_ and two entries for _Svc_A_ and _Svc_B_ respectively that each refer to the entry for _NetShare_ and with a unit _VolPath_ for each.
+1. "DefaultLabel" - If a label isn't specified at creation time of a volume for _nfs/nfs4 type volumes then this entry will be used to select the location for the volume.
+
+# Volumes
+
+1. Provisioned as a folder on the named network FS
+1. /mnt/vmdk/<remote—dir-label>/<vol-path>/<vol-name>/
+1. Volume _create_ mkdir’s the volume name folder if not already present
+
+## Volume meta-data
+Plugin maintains no in-memory state for either volumes or remote dirs. (except the configuration). Volume met-data listed below are generated when requested via a Get() from Docker,
+1. _Location_ – remote-dir, path within remote-dir
+1. _Type and version_ - volume type (NFS, EFS)
+1. _Capacity used in FS blocks_
+1. _Current usage count-
+1. _Create date/time_
+1. _Modified time_
+1. _Access time_
+1. _Access permissions_
+1. _Use count_
+
+
+# Volume options and Tenancy
+
+## Volume create options
+   - _access perms (ro, rw)_, remote-dir label, type (_file_ only allowed, if not specified defaults to _VMDK_ type)
+
+## Tenancy
+There is no support for tenancy as supported for VMDK backed volumes. Instead, the admin manages multi-tenancy. User controls which VMs have access to what datastores to support multiple tenants. Remote dirs configuration s available to help the admin determine what datastores are visible per docker host and where volumes get placed.
```

Comment:
"User controls which VMs have access to what datastores..."
Do you mean "NFS datastores" here?